### DISCUSSION TITLE
Disable "ready" and "unready" sounds coming from other players

### DISCUSSION THREAD

**Discussion by FoomZz** [ppy/osu#15788] [Ideas]
The ready/unready button in Lazer multiplayer lobbies can be spammed, and or used to troll other players. See the video below for an example. My proposal would to keep it like stables, where a sound is only played when everyone in the lobby is ready.

https://user-images.githubusercontent.com/63439048/143333380-e4a85b30-dcfd-496a-b521-9ec272a3d86b.mp4

**Comment by peppy** [comment]
This is intended for now. We may dim the volume in cases it is spammed, but it won't be disabled.

**Reply by FoomZz** [reply]
I'd take that over nothing. Can imagine this being quite annoying with bigger lobbies.
